bin/git-sed

```diff
@@ -64,6 +66,21 @@ actual command:
     shift
 done
 
-command="git grep -lz '$search' | xargs -0 sed -i 's/$search/$replacement/$flags'"
-git grep -lz "$search" | xargs -0 sed -i "s/$search/$replacement/$flags"
+all="$search$replacement$flags"
+case "$all" in
+    */*)
+        ascii="$(for((i=32;i<=127;i++)) do printf '%b' "\\$(printf '%03o' "$i")"; done)"
+        sep="$(printf '%s' "$ascii" | tr -d "$all")"
+        sep="$(printf %.1s "$sep")"
+        if [ "X$sep" = "X" ] ; then
+            echo 'could not find an unused character for sed separator character'
+            exit 1
+        fi
+        ;;
+    *)
+        sep=/
+        ;;
+esac
+command="git grep -lz '$search' | xargs -0r sed -i 's$sep$search$sep$replacement$sep$flags'"
```

Comment:
`-r` option is a GNU extension. The `xargs` in OS X doesn't support it.
We have to remove it for the compability.